Example/SlideController/MainController.swift

```diff
@@ -44,10 +60,34 @@ class MainController {
             break
         }
     }
+    
+    lazy var removeAction: (() -> Void)? = { [weak self] in
+        guard let strongSelf = self else { return }
+        strongSelf.slideController.removeAtIndex(index: 0)
+    }
+    
+    lazy var insertAction: (() -> Void)? = { [weak self] in
+        guard let strongSelf = self else { return }
+        let page = SlidePageModel<PageLifeCycleObject>(object: PageLifeCycleObject())
+        let index = strongSelf.slideController.content.count == 0 ? 0 : strongSelf.slideController.content.count - 1
+        strongSelf.slideController.insert(object: page, index: index)
+        if index < strongSelf.slideController.content.count {
```

Comment:
Looks like it's always true because object was inserted to slideController one line above